packages/schematics/angular/service/index_spec.ts

```diff
@@ -42,20 +42,21 @@ describe('Pipe Schematic', () => {
     expect(files.indexOf('/src/app/foo/foo.service.ts')).toBeGreaterThanOrEqual(0);
   });
 
-  it('should not be provided by default', () => {
-    const options = { ...defaultOptions };
+  it('service should be tree-shakeable', () => {
+    const options = { ...defaultOptions};
 
     const tree = schematicRunner.runSchematic('service', options, appTree);
-    const content = getFileContent(tree, '/src/app/app.module.ts');
-    expect(content).not.toMatch(/import { FooService } from '.\/foo\/foo.service'/);
+    const content = getFileContent(tree, '/src/app/foo/foo.service.ts');
+    expect(content).toMatch(/providedIn: 'root',/);
   });
 
-  it('should import into a specified module', () => {
+  it('should import a specified module', () => {
     const options = { ...defaultOptions, module: 'app.module.ts' };
 
     const tree = schematicRunner.runSchematic('service', options, appTree);
-    const content = getFileContent(tree, '/src/app/app.module.ts');
-    expect(content).toMatch(/import { FooService } from '.\/foo\/foo.service'/);
+    const content = getFileContent(tree, '/src/app/foo/foo.service.ts');
```

Comment:
Please replace usages of `getFileContent(tree, 'foo')` to `tree.readContent('foo')`. This means defining the type of `tree` to be `UnitTestTree` as `readContent` is a test helper method in a better place.

Reply:
Done.